### PR TITLE
[feature]add knife4j to linkis services(issue 1319)

### PR DESCRIPTION
**Pull request by aleneZeng**
### What is the purpose of the change
add knife4j to linkis  so that user can use apiDoc during development and debugging code.

### Brief change log
(for example:)
1,add maven depedency on knife4j 2.0.9
2,add knife4jConfig class
3,add related config param in application-linkis.yml

### Verifying this change
(Please pick either of the following options)  
This change is a trivial rework / code cleanup without any test coverage.  

### Does this pull request potentially affect one of the following parts:
- Dependencies (does it add or upgrade a dependency): yes
- Anything that affects deployment: no
- The MGS(Microservice Governance Services), i.e., Spring Cloud Gateway, OpenFeign, Eureka.: no

### Documentation
- Does this pull request introduce a new feature? no
- If yes, how is the feature documented? JavaDoc in class file Knife4jConfig

### CODE REVIEW

File: linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/Knife4jConfig.scala [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import org.springframework.beans.factory.annotation.Value
+
+/**
+ * it is very easy to enable knife when you want to use apiDoc(based on swagger2)
+ * you can follow these steps to enable
+ * 1, open application-linkis.yml and set knife4j.production=false
+ * 2, open linkis.properties and set ds.linkis.test.mode=true
+ * 3, restart the service and you can visit http://ip:port/api/rest_j/v1/doc.html
+ *
+ * or you can use apidoc by following steps  without enable ds.linkis.test.mode
```
Comment (peacewong) [inline linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/Knife4jConfig.scala:44]: “ds.linkis.test.mode”  the `wds` prefix will be removed in subsequent versions, and the references here still need to be correct ‘wds.linkis.test.mode'

Comment (aleneZeng) [reply linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/Knife4jConfig.scala]: correct the typo

Comment (peacewong) [reply linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/Knife4jConfig.scala]: ok,thanks

Comment (peacewong) [reply linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/Knife4jConfig.scala]: ok,thanks

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**